Jjdzr10 bd 16

filip-piatkowski opened the pull request:
Widok szczegółów pojedyńczej trasy.

filip-piatkowski committed:
Zmiana w track.html podświetlanie hover, dodanie pliku track-details.html i dodanie getTrackDetails w TrackController.

filip-piatkowski committed:
Dodanie sprawdzania nie istniejącej trasy i strony htlm z błędem.

filip-piatkowski committed:
Zmiana w FilePathConstants, na pliki w folderze wbowym.
Podmiana języka na polski w track-details.html

JGrotha commented on BugDoom - web app/pom.xml:
```diff
@@ -21,20 +21,30 @@
+			<groupId>org.thymeleaf.extras</groupId>
+			<artifactId>thymeleaf-extras-java8time</artifactId>
+			<version>3.0.4.RELEASE</version>
+		</dependency>
```
Nie potrzebujemy tej zależności

filip-piatkowski replied:
Poprawione

JGrotha commented on BugDoom - web app/src/main/resources/templates/tracks/track-error.html:
```diff
@@ -0,0 +1,19 @@
+<body>
+<div class="container mt-5">
+  <div class="alert alert-danger" role="alert">
+    Wybrany wyścig nie istnieje. Proszę spróbuj ponownie.
```
wyścig -> trasa

filip-piatkowski replied:
Poprawione.

grzeslukasik commented on BugDoom/pom.xml:
proponuję w templates utworzyć podfolder tracks i tam wrzucić track-detalis, track-error, track

filip-piatkowski replied:
Poprawione.

szablewski reviewed (commented):
jak poprawisz uwagi kolegów to będzie ok

filip-piatkowski committed:
Usuniecie nie potrzebnej zaleznosci w pom'ie.
Zamiana drobnych bledow jezykowych. Przeniesienie html plikow track do osobnego podfolderu w templates.

filip-piatkowski reviewed (commented):
Zamiany poprawione i wypushowane.

filip-piatkowski committed:
Merge branch 'develop' into JJDZR10BD-16

szablewski reviewed (approved):
ok

Commit:
Resolved conflicts with develop
